net-misc/asterisk-g729/asterisk-g729-13.0.3.1.9.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit multilib
+
+DESCRIPTION="G.729 codec and supporting files for asterisk"
+HOMEPAGE="https://store.digium.com/productview.php?product_code=G729CODEC"
```

Comment:
This seems to redirect to a new address.

Reply:
You're right.  That's new.